<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>NormalDialog</class>
    <widget class="QDialog" name="NormalDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>338</width>
                <height>228</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>30</y>
                    <width>271</width>
                    <height>111</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>Your Mean:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="meanBox"/>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                            <widget class="QLabel" name="label_2">
                                <property name="text">
                                    <string>Your Std Dev:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="devBox"/>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>140</x>
                    <y>160</y>
                    <width>168</width>
                    <height>30</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                    <widget class="QPushButton" name="cancelButton">
                        <property name="text">
                            <string>Cancel</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="okButton">
                        <property name="text">
                            <string>Ok</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
